equinix/fabric_service_profile_schema.go

```diff
@@ -1,8 +1,6 @@
 package equinix
 
-import (
-	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
-)
+import "github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
```

Comment:
There are a few stylistic changes like this that distract from the refactoring; could you revert them?

Reply:
I reverted the stylistic changes.